### PR TITLE
fix(ui): Use opaque background for SelectAllNotice

### PR DESCRIPTION
**Pull request by vuluongj20**
The current background is yellow100, which is semi-transparent. When scrolled, the issues list is visible underneath the alert. To fix this, we can add an opaque background behind the alert.

Before:
Screen Shot 2021-12-22 at 9 49 37 AM

After:
Screen Shot 2021-12-22 at 9 49 09 AM

### CODE REVIEW

**Review by JonasBa — APPROVED** [review]

File: static/app/views/issueList/actions/index.tsx [hunk 1]
```diff
@@ -368,6 +369,16 @@ const SelectAllNotice = styled('div')`
...
 `;
 
+const SelectAllNoticeBackground = styled('div')`
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: -1;
+  background: ${p => p.theme.background};
```
Comment (evanpurkhiser) [inline static/app/views/issueList/actions/index.tsx:379]: 🤔 What if we set two backgrounds, one the `theme.background` and the other a linear gradient with the yellow color.

That way we don't need an extra element?

Comment (vuluongj20) [reply static/app/views/issueList/actions/index.tsx]: @evanpurkhiser Oh that's a great idea! Implemented as an optional `opaque` prop on <Alert /> in https://github.com/getsentry/sentry/pull/30821/commits/a68fdab55e0033a495f99974d59200a4fc4e597a.

File: static/app/components/alert.tsx [hunk 2]
```diff
@@ -13,17 +13,12 @@ type Props = {
   system?: boolean;
   expand?: React.ReactNode[];
   expandIcon?: React.ReactNode;
+  opaque?: boolean;
```
Comment (evanpurkhiser) [inline static/app/components/alert.tsx:16]: When would we ever want it to actually not be opaque? haha

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/components/alert.tsx (hunk 2 above)
Comment (vuluongj20) [reply static/app/components/alert.tsx:16]: imo, most of the time 😛 I prefer semi-transparent backgrounds cause they let the color from behind bleed through, resulting in alerts that blend in more with the UI surrounding them.

Here's a comparison - the first is opaque, the second is semi-transaprent:
Screen Shot 2022-01-03 at 4 09 37 PM

Comment (evanpurkhiser) [reply static/app/components/alert.tsx:16]: 👍

**Pull request merged**